pkg/i18n/dutch.go

```diff
@@ -1,5 +1,90 @@
 package i18n
 
 func dutchSet() TranslationSet {
-	return TranslationSet{}
+	return TranslationSet{
+		PruningStatus:              "vernietigen",
+		RemovingStatus:             "verwijderen",
+		RestartingStatus:           "herstarten",
+		StoppingStatus:             "stoppen",
+		RunningCustomCommandStatus: "Aangepast commando draaien",
+
+		RunningSubprocess:                          "subprocess draaien",
+		NoViewMachingNewLineFocusedSwitchStatement: "No view matching newLineFocused switch statement",
+
+		ErrorOccurred:                     "Er is iets fout gegaan! Zou je hier een issue aan willen maken: https://github.com/jesseduffield/lazydocker/issues",
+		ConnectionFailed:                  "connectie naar de docker client mislukt. Het zou kunnen dat je de docker client moet herstarten",
+		UnattachableContainerError:        "Container heeft geen ondersteuning voor vastmaken. Je zou de service met het '-it' argument kunnen draaien of stop dit in je `stdin_open: true, tty: true` docker-compose.yml",
+		CannotAttachStoppedContainerError: "Je kan niet een vastgemaakte container stoppen, je moet het eerst starten (dit kan je doen met de 'r' toets) (ja ik ben te leu om dat voor je te doen automatisch)",
+		CannotAccessDockerSocketError:     "Kan de docker socket niet bereiken: unix:///var/run/docker.sock\nDraai lazydocker als root of lees https://docs.docker.com/install/linux/linux-postinstall/",
+
+		Donate:  "Doneer",
+		Confirm: "Bevestigen",
+
+		Return:             "terug",
+		FocusMain:          "focus hooft panneel",
+		Navigate:           "navigeer",
+		Execute:            "voer uit",
+		Close:              "sluit",
+		Menu:               "menu",
+		Scroll:             "scroll",
+		OpenConfig:         "open de lazydocker configuratie",
+		EditConfig:         "verander de lazydocker configuratie",
+		Cancel:             "annuleren",
+		Remove:             "verwijder",
+		ForceRemove:        "geforceerd verwijderen",
+		RemoveWithVolumes:  "verwijder met volumes",
+		RemoveService:      "verwijder containers",
+		Stop:               "stop",
+		Restart:            "herstart",
+		Rebuild:            "herbouw",
+		Recreate:           "recreate",
```

Comment:
I need a bit of help for *recreate*.  
I can't think of a good word for this aside from *herbouwen* what we already used for rebuild.  
Google translate suggests *herscheppen* but that sounds a bit to religious to me

Reply:
How about "hercreëer"? It's as close to the English, and while it's been a while since I considered Dutch to be my main language it still *feels* okay to me.

Reply:
Thanks, i'll use that